src/commands/runtime/activation/list.js

```diff
@@ -62,14 +62,18 @@ class ActivationList extends RuntimeBaseCommand {
           },
           Kind: {
             get: (row) => {
-              return row.annotations.find((elem) => {
+              const { annotations } = row
```

Comment:
I'm not crazy about the dangling return statement here, I prefer to see all operations inside blocks, and not depend on JS to only execute the single next line. 
ie 
```
if ( cond ) {
 return
}
```
instead of `if (cond) return`
I find it clearer, and if (inevitably) someone else comes along and adds another statement, the block is already there.

Reply:
That said, If we go with the affirmative version, I think it is much clearer
```
get: (row) => {
  const { annotations } = row
  if(annotations && annotations.length > 0) {
    return annotations.find((elem) => {
      return (elem.key === 'kind')
    }).value
  }
  return ''
}
```